services/postprocessing/pkg/config/config.go

```diff
@@ -35,6 +35,7 @@ type Postprocessing struct {
 
 // Events combines the configuration options for the event bus.
 type Events struct {
+	Workers  int    `yaml:"workers" env:"POSTPROCESSING_EVENTS_WORKERS" desc:"The number of concurrent go routines that fetch events from the event queue." introductionVersion:"6.7"`
```

Comment:
Would you mind moving this to `Postprocessing` struct? It is not really event configuration, but rather a Postprocessing specific one. 

You also dont need changes in `postprocessing.go` and `server.go`